<launch>
	<!-- declare arguments with default values -->
    	<arg name="image_source_topic" default="/usb_cam/image_raw"/>
        <arg name="test_img" default="/home/nano/test.jpg"/>
      
	    <node pkg="localimg_test" type="localimg_test" name="localimg_test" args="" output="screen">
	    <param name="test_img" type="str" value="$(arg test_img)"/>	
	    <param name="image_source_topic" value="$(arg image_source_topic)"/>	
	</node>
</launch>
